qlib/kernel/arch/x86_64/signal.rs

```diff
@@ -1,122 +0,0 @@
-// Copyright (c) 2021 Quark Container Authors / 2018 The gVisor Authors.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This should go to the main, not aarch64 feature branch

Reply:
Restored deleted file.